### DISCUSSION TITLE
How to change border color when in floating mode.

### DISCUSSION THREAD

**Discussion by hiroi-p** [qtile/qtile#2573] [Q&A]
Hi! I am new to qtile and I love it so far!
My problem is that when I put a windows into floating mode with mod and mouse the border color changes to blue. I don't like it and it's really out of theme, and so far I didn't find a way to change it in configs and I couldn't get anything in the docs or on the internet on this. Anyone knows how to change it?

Thank you!

**Comment by m-col** [comment]
Hi @HiroiOwO, Your config will have a `floating_layout` variable which is a `layout.Floating`. There is some info on available options to pass to it here, including border colours: http://docs.qtile.org/en/latest/manual/ref/layouts.html#floating. Hope that helps!
